documentation/src/main/wiki/Adapter.md

```diff
@@ -28,39 +30,150 @@ is not going to provide any generic Action Adapters.
 
 For custom Knots we can introduce custom Adapter types. As far as Knots must follow [[Knot contract|Knot#how-does-it-work]],
 Adapters are coupled with Knot directly so they can define their custom request, response or 
-configuration. The communication between Knot and Adapter can be custom too. 
+configuration. The communication between Knot and Adapter can be custom too.
+
+Communication contract between [[Service Knot|ServiceKnot]] and [[Http Service Adapter|HttpServiceAdapter]] is defined by:
+- `AdapterRequest` input,
+- `AdapterResponse` output.
+
+#### Adapter Request
+The table below describes the communication model between Knot.x Service Knot and Service Adapters.
+
+| Name                        | Type           | Mandatory | Description  |
```

Comment:
In the description put some example values of the data, e.g.:
- path-  `/services/mock/first.json`
- method - `GET, PUT, etc.`
- etc.